nodes/PayloadValidator.js

```diff
@@ -0,0 +1,52 @@
+const clone = require('clone');
+
+const variablesToCheck = [
+  'logger.metadata.organization',
+  'payload.system.organization',
+  'event.event.organization',
+  'event.event.token.contents.organization'
+];
+
+module.exports = class PayloadValidator {
+  constructor(_before) {
+    const before = clone(_before);
+    const {
+      logger, 
+      payload: {
+        system: {
+          bot, conversationId, organization, region
+        }
+      }
+    } = before;
+    this.before = before;
+    this.logger = logger;
+    this.bot = bot;
+    this.conversationId = conversationId;
+    this.organization = organization;
+    this.region = region;
+  }
+
+  verify(after) {
+    try {
+      variablesToCheck.forEach((location) => {
+        const getValue = (object) => location.split('.').reduce((p, c) => (p && p[c]) || null, object);
```

Comment:
can this be in the global scope?

Reply:
Sure, just done it here so didn't have to pass around the location and object.